modules/core/04-channel/keeper/upgrade.go

```diff
@@ -634,25 +635,23 @@ func (k Keeper) ChanUpgradeTimeout(
 	ctx sdk.Context,
 	portID, channelID string,
 	counterpartyChannel types.Channel,
-	prevErrorReceipt *types.ErrorReceipt,
-	proofCounterpartyChannel,
-	proofErrorReceipt []byte,
```

Comment:
If these args are no longer needed they can also be removed from `MsgChannelUpgradeTimeout`, right?

It would be good to also revise `ValidateBasic` of each of the msgs

Reply:
yeah for sure, I'd say we can do this in a follow up and clean up all the messages

Reply:
wanna create an issue? :)

Reply:
https://github.com/cosmos/ibc-go/issues/4450